trace.go

```diff
@@ -0,0 +1,11 @@
+package runn
+
+type trace struct {
+	RunID string `json:"id"`
+}
+
+func NewTrace(s *step) trace {
```

Comment:
Might not have to export.

```suggestion
func newTrace(s *step) trace {
```